.github/workflows/build.yml

```diff
@@ -31,6 +31,32 @@ jobs:
       - name: leanpkg build
         run: leanpkg build | python scripts/detect_errors.py
 
+      - name: configure git setup
+        run: |
+          git remote add origin-bot "https://$DEPLOY_NIGHTLY_GITHUB_USER:${{ secrets.DEPLOY_NIGHTLY_GITHUB_TOKEN }}@github.com/leanprover-community/mathlib.git"
+          git config user.email "leanprover.community@gmail.com"
+          git config user.name "leanprover-community-bot"
+          git config --unset http.https://github.com/.extraheader
+
+      - name: push release to azure
+        if: always() && github.repository == 'leanprover-community/mathlib' && github.event_name == 'push' && matrix.lean_version == 'leanprover-community/lean:3.5.1'
+        run: |
+          git fetch --depth=1 origin-bot master
+          remote_sha="$(git rev-parse origin-bot/master)"
```

Comment:
```suggestion
          remote_sha="$(git rev-parse origin-bot/SOME_VARIABLE_FOR_THE_CURRENT_BRANCH)"
```

Reply:
I struggled to get the branch name from Actions in a way that's compatible with `git rev-parse`. This seems to work but I'm not sure it's optimal.

Reply:
I struggled to get the branch name from Actions in a way that's compatible with `git rev-parse`. This seems to work but I'm not sure it's optimal.